app/models/concerns/sufia/solr_document_behavior.rb

```diff
@@ -7,6 +7,14 @@ module SolrDocumentBehavior
     include Sufia::SolrDocument::Export
     include Sufia::SolrDocument::Characterization
 
+    def all_titles
```

Comment:
Did you first try to name this `title`, thereby overriding CC's method of the same name? Or is there value in having both methods defined on Solr documents?